backend/processing/check_flat_directory_structure/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04
```

Comment:
This file is identical to the other Dockerfile. Can we use the same Dockerfile for all of the processing steps?

Reply:
Yes it is the same, but currently I don't see any configuration in [aws_batch.JobDefinition](https://docs.aws.amazon.com/cdk/api/latest/python/aws_cdk.aws_batch/JobDefinition.html) which will allow us to use single Docker file